test/functional/test_framework/address.py

```diff
@@ -158,6 +158,11 @@ def check_script(script):
         return script
     assert False
 
+def bech32_to_bytes(address):
```

Comment:
styling nit: two blank lines between functions, per PEP8 guidelines

Reply:
I think for consistency in the code base it's okay.
There is only one blank line between functions in address.py

Reply:
The goal is to have the code base consistent with our style guide, not consistent with itself. This means any new code should adhere to the style guide and, if possible, old code should be formatted consistent with the style guide if it's already being touched as part of a PR.